### PR TITLE
Randomize characters and skills in load tests

### LINKED ISSUE #397: Randomize characters and skills in load tests
We need to create mocked matches as real as possible.

### PR DESCRIPTION
**Pull request by Nico-Sanchez**
## Motivation
LoadTest's mocked players were all h4cks and they were using their basic skill only (we got ulti and dash too).
Closes #397

## Changes:

- [Set random character instead of always being h4ck](https://github.com/lambdaclass/mirra_backend/commit/e9c055e84bda6c6e223fcf9ba9c5cbbaf0d2e371)
- [Make mocked players to use random skills](https://github.com/lambdaclass/mirra_backend/commit/3d455b421d5bcc080c7bacd029a10cd4229ab670)
- [Make valtimer inactive because he's not active in the main client](https://github.com/lambdaclass/mirra_backend/commit/35342c1bf393fd785bc521f41717bd9f6b98d4d8)
- [Remove TODO tag message](https://github.com/lambdaclass/mirra_backend/commit/f0adebf594f326c5653dfc1a301482d559ceaff1)

## How to test it?

I'll attach a video about it. Basically what you do is start a game and instantiate as many mocked players as you want.

https://github.com/lambdaclass/mirra_backend/assets/65268603/402cbb9a-57e3-45b6-9d3b-9dfecd7717bf

### CODE REVIEW

**Comment by agustinesco** [comment]
There's a warning when you compile the application

```
     warning: Arena.Configuration.get_game_config/0 is undefined (module Arena.Configuration is not available or is yet to be defined)
     \x{2502}
 111 \x{2502}     Arena.Configuration.get_game_config()
     \x{2502}                         ~
     \x{2502}
     \x{2514}\x{2500} (arena_load_test 0.1.0) lib/arena_load_test/game_socket_handler.ex:111:25: ArenaLoadTest.GameSocketHandler.get_available_skills/0

    warning: Arena.Configuration.get_game_config/0 is undefined (module Arena.Configuration is not available or is yet to be defined)
    \x{2502}
 72 \x{2502}     Arena.Configuration.get_game_config()
    \x{2502}                         ~
    \x{2502}
    \x{2514}\x{2500} (arena_load_test 0.1.0) lib/arena_load_test/socket_handler.ex:72:25: ArenaLoadTest.SocketHandler.get_random_active_character/0
```

**Comment by Nico-Sanchez** [comment]
> There's a warning when you compile the application, the load testing is working perfect tho
> 
> ```
>      warning: Arena.Configuration.get_game_config/0 is undefined (module Arena.Configuration is not available or is yet to be defined)
>      \x{2502}
>  111 \x{2502}     Arena.Configuration.get_game_config()
>      \x{2502}                         ~
>      \x{2502}
>      \x{2514}\x{2500} (arena_load_test 0.1.0) lib/arena_load_test/game_socket_handler.ex:111:25: ArenaLoadTest.GameSocketHandler.get_available_skills/0
> 
>     warning: Arena.Configuration.get_game_config/0 is undefined (module Arena.Configuration is not available or is yet to be defined)
>     \x{2502}
>  72 \x{2502}     Arena.Configuration.get_game_config()
>     \x{2502}                         ~
>     \x{2502}
>     \x{2514}\x{2500} (arena_load_test 0.1.0) lib/arena_load_test/socket_handler.ex:72:25: ArenaLoadTest.SocketHandler.get_random_active_character/0
> ```

Opps. I hardoded this since we'll take them from the bots app later.

**Review by agustinesco — DISMISSED** [review]
LGTM!

File: apps/arena_load_test/lib/arena_load_test/game_socket_handler.ex [hunk 1]
```diff
@@ -104,4 +104,11 @@ defmodule ArenaLoadTest.GameSocketHandler do
         "ws://#{host}/play/#{game_id}/#{client_id}"
     end
   end
+
+  # This is enough for now. We will get the skills from the requested bots
+  # from the bots app. This will be done in future iterations.
```
Comment (lotuuu) [inline apps/arena_load_test/lib/arena_load_test/game_socket_handler.ex:109]: Is there an issue for this already?

File: apps/arena_load_test/lib/arena_load_test/socket_handler.ex [hunk 2]
```diff
@@ -69,4 +67,10 @@ defmodule ArenaLoadTest.SocketHandler do
         "ws://#{host}/join/#{player_id}/#{character}/#{player_name}"
     end
   end
+
+  # This is enough for now. Will request bots from the bots app in future iterations.
```
Comment (manucamejo) [inline apps/arena_load_test/lib/arena_load_test/socket_handler.ex:71]: Please create an issue for this

Comment (Nico-Sanchez) [reply apps/arena_load_test/lib/arena_load_test/socket_handler.ex:71]: Done.

File: apps/arena_load_test/lib/arena_load_test/game_socket_handler.ex (hunk 1 above)
Comment (Nico-Sanchez) [reply apps/arena_load_test/lib/arena_load_test/game_socket_handler.ex:109]: Created one and linked it here.

**Review by manucamejo — APPROVED** [review]

**Review by agustinesco — APPROVED** [review]

**Pull request merged**